src/message_interface/mod.rs

```diff
@@ -261,4 +261,69 @@ mod tests {
         }
         std::fs::remove_dir_all("test-storage/message_interface_stronghold").unwrap_or(());
     }
+
+    #[tokio::test]
+    async fn address_conversion_methods() {
+        std::fs::remove_dir_all("test-storage/address_conversion_methods").unwrap_or(());
+        let secret_manager = r#"{"Mnemonic":"acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast"}"#.to_string();
+        let client_options = r#"{
+            "nodes":[
+               {
+                  "url":"http://localhost:14265/",
+                  "auth":null,
+                  "disabled":false
+               },
+               {
+                  "url":"https://chrysalis-nodes.iota.cafe/",
+                  "auth":null,
+                  "disabled":false
+               }
+            ],
+            "localPow":true,
+            "apiTimeout":{
+               "secs":20,
+               "nanos":0
+            }
```

Comment:
This is a little bit useless for this test, just 
````JSON
{
            "nodes":["http://localhost:14265/"]
}
````
or very similar should also work